README.md

```diff
@@ -78,3 +78,18 @@ A VSCode debug config is available to run and debug the project.
 
 To make the exporter talk with a Ceph RGW S3 endpoint, create a copy of the `.env.example` file and name it `.env`.
 Be sure ot add your Ceph RGW S3 endpoint and credentials in it.
+
+### Create a new Release
+
+To create a new release (example is for release `v0.1.2`):
+
+1. Increase the version according to Semantic Versioning in the [`VERSION` file](VERSION).
+2. Add a new entry to the [`CHANGELOG.md`](CHANGELOG.md) with the changes and improvements listed in it.
```

Comment:
Do we also need to uodate [charts/extended-ceph-exporter/Chart.yaml](charts/extended-ceph-exporter/Chart.yaml)?